src/actions/languages/python/motion.ts

```diff
@@ -0,0 +1,256 @@
+import { RegisterAction } from '../../base';
+import { VimState } from '../../../state/vimState';
+import { BaseMovement } from '../../baseMotion';
+import { Position, TextDocument } from 'vscode';
+
+interface LineInfo {
+  line: number;
+  indentation: number;
+  text: string;
+}
+
+interface MinimalPosition {
```

Comment:
This seems unnecessary - why not just use `vscode.Position`?

Reply:
Good catch. Removed.